session.go

```diff
@@ -0,0 +1,113 @@
+package scylla
+
+import (
+	"fmt"
+	"net"
+	"time"
+
+	"scylla-go-driver/transport"
+)
+
+// TODO: Should we support connecting to different than default port?
+// TODO: Add retry policy.
+// TODO: Add Query Paging.
+// TODO: make better randomConnection.
+
+type SessionConfig struct {
+	hosts []string
+	transport.ConnConfig
+}
+
+func MakeSessionConfig(tcpNoDelay bool, defaultConsistency uint16, hosts ...string) SessionConfig {
+	cpy := make([]string, len(hosts))
+	copy(cpy, hosts)
+
+	return SessionConfig{
+		hosts: cpy,
+		ConnConfig: transport.ConnConfig{
+			TCPNoDelay:         tcpNoDelay,
+			Timeout:            5 * time.Second, // Should user have control of timeout value?
+			DefaultConsistency: defaultConsistency,
+		},
+	}
+}
+
+// I don't know if we need this.
+func discoverHosts(hosts []string) []net.IP { // nolint:deadcode,unused // This might be used in the future.
```

Comment:
As discussed we want to use the host names (DNS) down to the final dial.
Please remove.